spec/features/idv/doc_auth/document_capture_step_spec.rb

```diff
@@ -238,6 +238,53 @@
         expect(page).to have_content(I18n.t('errors.doc_auth.acuant_network_error'))
       end
     end
+
+    context 'when there is a stored result' do
+      it 'proceeds to the next step if the result was successful' do
+        document_capture_session = user.document_capture_sessions.last
+        response = DocAuthClient::Response.new(success: true)
+        document_capture_session.store_result_from_response(response)
+        document_capture_session.save!
+
+        page.driver.put current_path
```

Comment:
This is a hacky way to simulate the frontend submitting an empty form here.

Reply:
Maybe clarify that by naming it with a helper method?

```
def submit_empty_form
  page.driver.put current_path
end
```